src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/UnitTestingIncrementalAnalyzer.cs

```diff
@@ -15,8 +17,8 @@ internal class UnitTestingIncrementalAnalyzer : IIncrementalAnalyzer
         public UnitTestingIncrementalAnalyzer(IUnitTestingIncrementalAnalyzerImplementation implementation)
             => _implementation = implementation;
 
-        public Task AnalyzeDocumentAsync(Document document, SyntaxNode bodyOpt, InvocationReasons reasons, CancellationToken cancellationToken)
```

Comment:
Are these renames in the external access project not a breaking change?

Reply:
Only the code in the [Api namespace](https://github.com/dotnet/roslyn/tree/a2d7737ebd3ed9896d1a2880d8c56335f9a6c7c2/src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api) is externally visible.